libnavui-base/src/test/java/com/mapbox/navigation/ui/base/model/maneuver/DelimiterComponentNodeTest.kt

```diff
@@ -0,0 +1,22 @@
+package com.mapbox.navigation.ui.base.model.maneuver
+
+import com.mapbox.navigation.testing.BuilderTest
+import org.junit.Test
+import kotlin.reflect.KClass
+
+class DelimiterComponentNodeTest : BuilderTest<DelimiterComponentNode,
```

Comment:
Should we name it `DelimiterComponentNodeBuilderTest`? Same for the rest of the `Builder`s below 👇

Reply:
I would like to keep the name of the test file the same as the name of the class file.